Framework/Algorithms/src/PolarizationCorrections/HeliumAnalyserEfficiency.cpp

```diff
@@ -103,20 +103,15 @@ void HeliumAnalyserEfficiency::init() {
  */
 std::map<std::string, std::string> HeliumAnalyserEfficiency::validateInputs() {
   std::map<std::string, std::string> errorList;
-  const std::string inputWorkspaceName = getProperty(PropertyNames::INPUT_WORKSPACE);
-  if (!AnalysisDataService::Instance().doesExist(inputWorkspaceName)) {
-    errorList[PropertyNames::INPUT_WORKSPACE] =
-        "The input workspace " + inputWorkspaceName + " does not exist in the ADS.";
+  WorkspaceGroup_sptr ws = getProperty(PropertyNames::INPUT_WORKSPACE);
+
+  if (!ws->isGroup()) {
```

Comment:
The `WorkspaceGroup` input seems to offer a lot of built in validation so this doesn't actually cause an issue in practice, but in many cases when retrieving property values the code on line 106 would return a `nullptr` if something was passed in that couldn't be converted to a `WorkspaceGroup_sptr`. So the check we want to put here is: `if(ws == nullptr)`.